js/pages/characterizations/services/conversion/BaseStatConverter.js

```diff
@@ -50,7 +50,7 @@ define([
             analysis.reports.forEach((r, i) => r.stats.forEach(mapCovariate(r)));
 
             cohorts.forEach((c, i) => {
-              for (let strataId of strataNames.keys()) {
+              for (let strataId of Array.from(strataNames.keys()).sort()) {
```

Comment:
Shouldn't
- the values be either sorted inside map
or
- the sorting be repeated inside the template
?